frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetails.tsx

```diff
@@ -236,11 +237,29 @@ export function PolicyTemplateDetails(
               )
               break
             default:
-              compliant = (
-                <div>
-                  <ExclamationTriangleIcon color="var(--pf-global--warning-color--100)" /> {t('No status')}
-                </div>
-              )
+              switch (item.object?.kind) {
```

Comment:
Could you make this switch statement only run if the policy is an OperatorPolicy?